oam/install_scripts/syslogSetup.sh

```diff
@@ -156,19 +160,26 @@ install() {
 checkSyslog
 if [ ! -z "$syslog_conf" ] ; then
 	$installdir/bin/setConfig -d Installation SystemLogConfigFile ${syslog_conf} >/dev/null 2>&1
-	if [ "$syslog_conf" != /etc/rsyslog.d/columnstore.conf ]; then
-		$SUDO rm -f ${syslog_conf}.columnstoreSave
-		$SUDO cp ${syslog_conf} ${syslog_conf}.columnstoreSave >/dev/null 2>&1
-		$SUDO sed -i '/# MariaDB/,$d' ${syslog_conf}.columnstoreSave > /dev/null 2>&1
+	if [ $user != "root" ]; then
+		 chown $user:$user /home/$user/mariadb/columnstore/etc/*
```

Comment:
first arg should probably be $username:$groupname.  Also, use $HOME to refer to the user's home, don't assume it's under /home/.  But also, they should already own that dir b/c they're the ones that installed it.